arkouda/pdarrayclass.py

```diff
@@ -205,6 +205,8 @@ def _binop(self, other : pdarray, op : str) -> pdarray:
             if self.size != other.size:
                 raise ValueError("size mismatch {} {}".format(self.size,other.size))
             cmd = "binopvv"
+            if self.ndim != 1:
+                cmd += "2d"
```

Comment:
I am wondering if it would be better to check specifically for a dimension of 2 here. This shouldn't be an issue now, since we are only expecting dimensions of 1 or 2, but could help identify anything out of the oridinary with the dimension in the future.